pkg/users/local.js

```diff
@@ -31,6 +31,20 @@
 
 superuser.reload_page_on_change();
 
+export const admins = ['sudo', 'root', 'wheel'];
+const sortGroups = groups => {
+    return groups.sort((a, b) => {
+        if (a.isAdmin)
+            return -1;
+        if (b.isAdmin)
+            return 1;
+        if (a.members === b.members)
+            return a.name.localeCompare(b.name);
+        else
+            return b.members - a.members;
+    });
+};
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17996-20221212-195940-2c392d36-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)